slo_compliance.py

```diff
@@ -0,0 +1,111 @@
+
+
+def updated(days):
+    return 'updated:<={}'.format(today-timedelta(days))
+
+
+def created(days):
+    return 'created:<={}'.format(today-timedelta(days))
+
+
+def query(url, params, name):
+    r = requests.get(url, params=params)
+    soup = BeautifulSoup(r.text, 'html.parser')
+    number = int(soup
+                 .find('a', class_='btn-link selected')
+                 .contents[2]
+                 .strip()[:-5])
+    items = soup.find_all('span', class_='opened-by')
+    print(r.url)
+    print(name, number)
+    print([int(item.contents[0].strip().split("\n")[0][1:]) for item in items])
+
+
+issues = {'issues_with_no_type': (no+types.bug
+                                  + no+types.feature
+                                  + no+types.question
+                                  + no+types.cleanup
+                                  + no+types.process),
+          'questions_with_no_assignee': 'no:assignee ',
+          'bugs_with_no_assignee': types.bug + 'no:assignee',
+          'bugs_with_no_priority': (types.bug
+                                    + no+priority.p0
+                                    + no+priority.p1
+                                    + no+priority.p2),
+          'bugs_with_priority_p0': types.bug + priority.p0,
+          'bugs_with_priority_p1': types.bug + priority.p1,
+          'bugs_with_priority_p2': types.bug + priority.p2,
+          'questions_outside_response_slo': types.question,
+          'p0_bugs_outside_response_slo': (types.bug
+                                           + priority.p0
+                                           + updated(1)),
+          'p0_bugs_outside_closure_slo': (types.bug
+                                          + priority.p0
+                                          + created(5)),
+          'p1_bugs_outside_response_slo': (types.bug
+                                           + priority.p1
+                                           + updated(5)),
+          'p1_bugs_outside_closure_slo': (types.bug
+                                          + priority.p1
+                                          + created(42)),
+          'p2_bugs_outside_response_slo': (types.bug
+                                           + priority.p2
+                                           + updated(120)),
+          'p2_bugs_outside_closure_slo': (types.bug
+                                          + priority.p2
+                                          + created(12)),
+          'feature_requests_outside_response_slo': (types.feature
+                                                    + updated(120)),
+          'feature_requests_outside_closure_slo': (types.feature
+                                                   + created(120)),
+          'cleanup_outside_response_slo': (types.cleanup
+                                           + updated(120)),
+          'cleanup_outside_closure_slo': (types.cleanup
+                                          + created(120))}
```

Comment:
As far as I know, there is no SLO for closure / response to a `type: cleanup` issue.

Reply:
I think there isn't an official one, but if it's been there for 4 months, maybe we should close it out or do something with it?
ditto for pull requests.